cmd/wksctl/profile/enable/enable.go

```diff
@@ -78,6 +82,23 @@ func profileEnableRun(params profileEnableFlags) error {
 	}
 	log.Info("Added the profile to the local repository.")
 
+	ignoreFilePath := path.Join(clonePath, constants.WKSctlIgnoreFilename)
+	if _, err := os.Stat(ignoreFilePath); err == nil {
+		log.Info("Ignoring files declared in .wksctlignore...")
+		file, err := os.Open(ignoreFilePath)
```

Comment:
Leaked fd. This `Open` should have a corresponding `Close`. If this whole block gets separated to a smaller function, the `Close` could be `defer`red.

Reply:
good catch! thank you.